client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -50,8 +51,6 @@ class RatingsAndReviews extends React.Component {
     Promise.all([this.getReviews(), this.getRatings()])
```

Comment:
This isn't directly related to this PR, but is there a way you could make this work without loading all reviews on every refresh? I'm getting kicked off the API sometimes for "too many requests" when testing and I think this might have something to do with it.